rustls/src/conn/unbuffered.rs

```diff
@@ -267,6 +314,57 @@ impl<'c, 'i, Data> AppDataAvailable<'c, 'i, Data> {
     }
 }
 
+/// Application-data is available
```

Comment:
I think this should say "Early application-data is available" to distinguish from `AppDataAvailable` that already says "Application-data is available"